CHANGES.md

```diff
@@ -1,3 +1,62 @@
+# Version 0.5
+
+## New Features
+
+* Added experimental support for basic, non-compositional verification
+  of x86 machine code for use in conjunction with LLVM verification.
+
+        crucible_llvm_verify_x86 :
+          LLVMModule -> String -> String ->
+          [(String, Int)] -> Bool -> CrucibleSetup () ->
+          TopLevel CrucibleMethodSpec
+
+
+  The first argument specifies the LLVM module containing the _caller_.
+  The second and third specify the ELF file name and symbol name of the
+  function to be verifier. The fourth specifies the names and sizes (in
+  bytes) of global variables to initialize, and the fifth whether to
+  perform path satisfiability checking. The last argument is the LLVM
+  specification of the calling context against which to verify the
+  function.
+
+* Added support for using the SMT theory of arrays in the LLVM memory
+  model. In some cases, this can significantly improve performance.
+
+        enable_smt_array_memory_model : TopLevel ()
+        disable_smt_array_memory_model : TopLevel ()
+
```

Comment:
* add `crucible_alloc_aligned` and `crucible_alloc_readonly_aligned`
* add `crucible_conditional_points_to`

Reply:
Added.